ietf-network-slice.yang

```diff
@@ -1081,6 +1113,14 @@ module ietf-network-slice-service {
         }
       }
     }
+    leaf slo-sle-policy-override {

```

Comment:
Looking at the comment in the issue #26  https://github.com/lana-wu/ietf-ns-nbi/issues/26#issuecomment-1175029923
it seems that you can have 3 different values of identity managing inheritance but only one attribute has been defined covering "override" option only. Is that the intention and if yes, it seems in contrddiction with the explanation contained in the comment I've reported above. It would be beneficial anyway to have some text clarifying the inheritance management.